playbooks/roles/edx_django_service/defaults/main.yml

```
@@ -102,6 +102,7 @@ edx_django_service_basic_auth_exempted_paths_extra: []
 edx_django_service_basic_auth_exempted_paths: '{{ edx_django_service_basic_auth_exempted_paths_default + edx_django_service_basic_auth_exempted_paths_extra }}'
 
 edx_django_service_newrelic_appname: '{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-{{ edx_django_service_name }}'
+edx_django_service_disable_newrelic_app: false
```

Since we have the luxury of setting a default, I'd prefer fewer negatives:

```suggestion
edx_django_service_enable_newrelic_app: true
```